util/packaging/homebrew/chapel-release.rb

```diff
@@ -1,34 +1,51 @@
 class Chapel < Formula
   desc "Programming language for productive parallel computing at scale"
   homepage "https://chapel-lang.org/"
-  url "https://github.com/chapel-lang/chapel/releases/download/1.26/chapel-1.26.0.tar.gz"
+  url "https://github.com/chapel-lang/chapel/releases/download/1.26.0/chapel-1.26.0.tar.gz"
```

Comment:
ummmmmm... shouldn't this be using 1.27 ? I was expecting this file to be a copy of https://github.com/Homebrew/homebrew-core/pull/104773/files ... and at this point in the release cycle, the same as chapel-main.rb.